saleor/graphql/query_cost_map.py

```diff
@@ -299,10 +387,6 @@
     "SaleChannelListing": {
         "channel": {"complexity": 1},
     },
-    "SelectedAttribute": {
```

Comment:
, this cost should be removed when the field is removed from the API, rather than when it is marked as deprecated.

Reply:
It is something remvoed by accident. Thanks :)